packages/callhierarchy/src/browser/callhierarchy-context.ts

```diff
@@ -70,8 +70,15 @@ export class CallHierarchyContext implements Disposable {
         }
         if (!locations) {
             return undefined;
+        } else if (Array.isArray(locations)) {
```

Comment:
looks bogus, `LocationLink` should be handled as well

proper solution is here: https://github.com/theia-ide/theia/pull/5901/files#diff-c52289f90104a8f73b07c6ab88357987R74

Reply:
Thanks, since it wasn't handled in this specific code before, I left it to a minimal working state. Will change.